ui/src/settings/components/ChangeEmailForm/ChangeEmailForm.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Button } from 'antd';
+import { Formik, Field, Form } from 'formik';
+import { object, string } from 'yup';
+
+import './ChangeEmailForm.less';
+import { Credentials } from '../../../types';
+import TextField from '../../../submissions/common/components/TextField';
+import EventTracker from '../../../common/components/EventTracker';
+
+const SCHEMA = object().shape({
+  email: string().email().trim().required().label('Email'),
+});
+
+export const ChangeEmailForm = ({
+  onChangeEmailAddress,
+  loading,
+  email,
+}: {
+  onChangeEmailAddress: ((credentials: Credentials) => void | Promise<any>) &
+    Function;
+  loading: boolean;
+  email?: string;
+}) => {
+  return (
+    <Formik
+      validationSchema={SCHEMA}
+      validateOnBlur
+      validateOnChange={false}
+      onSubmit={onChangeEmailAddress}
+      initialValues={{ email }}
+    >
+      {(props) => (
+        <Form>
+          <p>
+            Change the email address associated to your INSPIRE account. This is
+            used by the INSPIRE system to contact you automatically (e.g. job ad
+            closing, submission confirmation, etc.).
+          </p>
+          <div className="pt3 flex">
+            <div style={{ flex: 12 }}>
+              <Field
+                name="email"
+                type="email"
+                placeholder="Email"
+                data-test-id="email"
+                component={TextField}
+                className="w-100 mb0"
+              />
+            </div>
+            <EventTracker
+              eventCategory="Settings"
+              eventAction="Edit"
+              eventId="Change user password"
+            >
+              <Button
+                loading={loading}
+                disabled={!props.isValid || !props.dirty}
+                type="primary"
+                htmlType="submit"
+                data-test-id="submit-email"
+                style={{ flex: 1 }}
+              >
+                Change
+              </Button>
+            </EventTracker>
+          </div>
+        </Form>
+      )}
+    </Formik>
```

Comment:
@karolina-siemieniuk-morawska can we make the button to be next to the form? now it's on the other side
![Screenshot 2023-01-31 at 14 39 14](https://user-images.githubusercontent.com/4287772/215775798-5b94582a-934a-4690-af16-eb10acf4b13c.png)

Reply:
@drjova should the button be moved closer to the input or make the input field longer so that the box and button take up the whole line?

Reply:
I would say move the button next to the form